### ISSUE TITLE
13 peers found but still not syncing.

### ISSUE THREAD

**Issue by w1-denambtmanr** [ethereum/mist#3503]
I am stuck on the amount of blocks left but it does find 13 peers but it just doesn't download. Unless I restart mist over and over and over again just to get about ~5k blocks downloaded. If I have to do that I'm still busy for like a month. Anyone any clue?

```
Version: `0.9.3`
OS & Version: windows
Node version: `geth 1.7.2` 
Number of blocks synchronized: ?
```

<!--
![image](https://user-images.githubusercontent.com/10830858/34537643-7b86dcc8-f0c9-11e7-8643-9776e0d4f9b2.png)

**Comment by lowlanda** [comment]
It's broken, see issue #2466 and find post about MyEtherWallet, if you want/need an alternative solution

We've given up lol

**Comment by w1-denambtmanr** [comment]
Yeah I'm thinking about syncing for one last time then just send all the ETH over to a different wallet. Fuck this shit lol.
